### ISSUE TITLE
bunch of error in CameraViewController

### ISSUE THREAD

**Issue by murad1981** [eBay/NMessenger#188]
Argument of '#selector' refers to instance method 'capture' that is not exposed to Objective-C

there are many errors like this in CameraViewController, is there gonna be a release that has these errors fixed.

**Comment by dodikk** [comment]
@murad1981 , the project seems a abandoned by eBay and the PRs do not get merged for months.
Please take a look at this PR and its origin. Really sad...

I've fixed some camera related issues there.
https://github.com/eBay/NMessenger/pull/145

P.S. I've also added a few issues such as multiple selection of image attachments. Please let me know if that was helpful.

**Comment by murad1981** [comment]
yeah it sounds abandoned, I don't know why e-bay isn't caring about it anymore although it's a huge effort. I've checked your PR, thank you.
